spec/requests/internal/users_spec.rb

```diff
@@ -164,13 +163,12 @@ def call_ghost
 
   context "when deleting user" do
     def create_mention
-      comment = create(
+      create(
         :comment,
         body_markdown: "Hello @#{user.username}, you are cool.",
         user_id: user2.id,
         commentable_id: article2.id,
       )
-      Mention.create_all_without_delay(comment)
```

Comment:
We can't just delete these methods from the test, because it changes the behavior. This method still needs to create mentions, so you need to replace it with calling `Mentions::CreateAllJob.perform_now` or create mentions manually.

Reply:
We must to add a test that expects this behavior, because now without it tests pass

Reply:
There could be tests that check that all parts of user activity were deleted, but I wouldn't make it a part of this pr. For now, let it just check that when user has the specified types of activities, it gets deleted, and the request doesn't fail.